package.json

```diff
@@ -31,25 +31,29 @@
   "author": "Matteo Collina <hello@matteocollina.com>",
   "contributors": [
     {
+      "name": "Tommaso Allevi",
+      "email": "tomallevi@gmail.com"
+    },
+    {
```

Comment:
I would prefer for Tomas to be the 1st of the contributors, if you are ok with it.

Reply:
The wrong Tom :smile:

Reply:
I get it. I wish package.json supported more than a single author. It definitely needs more support for a large project developed by multiple people.